infrastructure/application/services/hasura.ts

```diff
@@ -149,17 +145,17 @@ export const createHasuraService = async ({
               name: "HASURA_PLANX_API_KEY",
               value: config.require("hasura-planx-api-key"),
             },
-            // extend timeout for migrations during setup to 10 mins (default is 30s)
+            // extend timeout for migrations during setup to 3 mins (default is 30s)
             {
               name: "HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT",
-              value: "600",
+              value: "180",
```

Comment:
This is ample time for the Hasura migrations to happen, if they're ever going to! Most of the edits to values above are just a tightening up which flow from this point.